test/integration/handlers/authorize-handler_test.js

```diff
@@ -278,8 +269,7 @@ describe('AuthorizeHandler integration', function() {
       return handler.handle(request, response)
         .then(function() {
           response.get('location').should.equal('http://example.com/cb?code=12345&state=foobar');
-        })
-        .catch(should.fail);
```

Comment:
Why was this removed?

Reply:
This test is not failing, so there is no point on catch an error.

Reply:
Fixed. 👍